src/org/opendatakit/briefcase/ui/export/ExportPanel.java

```diff
@@ -178,7 +181,9 @@ private void export() {
               terminationFuture
           );
         });
+    //notify UI
     form.enableUI();
+    form.hideExporting();
```

Comment:
We have to remove this call from here. Exporting happens asynchronously and this will get executed before the export process ends.